README.md

```diff
@@ -79,6 +87,127 @@ components:
       format: date-time
       description: Date and time when the message was sent.
 ```
+
+</p>
+</details>
+
+<details><summary>MQTT</summary>
+<p>
+
+```yml
+asyncapi: '2.0.0'
```

Comment:
is it the file you are testing mqtt support against?

Reply:
@derberg against very similar to this

Reply:
@Tenischev please keep in mind that we want to avoid testing on streetlights to minimul as we already produced so many different variations of it that it is no longer an example but a Frankenstein 😄 . Not sure if you recall the slack discussion, that we want to work as much as possible on a dummy-like example that we already started crafting in generator repo. Please keep that in mind and explore switching to dummy example